### DISCUSSION TITLE
All chunks reload when opening/closing chat

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#34] [Bugs]
Whenever you press forward slash to type a command, the entire world reloads

https://user-images.githubusercontent.com/26075577/223855885-2e0ff4ed-5826-4f9d-9a23-b980348b6902.mp4

**Comment by tryashtar** [comment]
Switching in and out of crosshair mode also does this

**Comment by minecraftchloe** [comment]
Yeah - this is a very known issue, and for chat, something we'll address in an alternate way by creating UI elements to support. For now, marking as `expected`, but as Editor evolves beyond v0.3 you'll start to see various things converted into UI panels.
